File: python/problema2.py
'''
Crie uma express ̃ao Lambda que receba trˆes notas de um aluno (a, b, c),
 calcule a m ́edia e retorne se oaluno foi aprovado ou reprovado.
   Para um aluno ser aprovado, ele deve possuir nota igual ou superior a6.
     Leia as notas dos alunos do teclado.
'''

m = []
for i in range(0,3):
    m.append(int(input("Digite uma nota\n")))
print("It's '" + str((lambda a,b,c : (a+b+c)/3 >= 6) (m[0],m[1],m[2])) + "' that the student was approved")

File: HaskelAtividades/HaskellBasicoPIV/problema2.hs
{-
2. Crie uma função que receba um número x, negativo ou positivo, e retorne seu valor absoluto. Leia x do teclado.
-}

absoluto :: Float -> Float
absoluto x = (x*x)**0.5

main = do
    xString <- getLine
    let x = (read xString :: Float)
    let absol = absoluto x
    print(absol)
